cli/src/validator_info.rs

```diff
@@ -223,9 +227,12 @@ impl ValidatorInfoSubCommands for App<'_, '_> {
                                 .value_name("PUBKEY")
                                 .takes_value(true)
                                 .validator(is_pubkey)
-                                .help("The pubkey of the Validator info account; without this argument, returns all"),
+                                .help(
+                                    "The pubkey of the Validator info account; without this \
+                                     argument, returns all",
```

Comment:
nit - I think what is being returned is implied, but sounds a little off reading it out loud. What do you think ?
```
argument, returns all Validator info accounts",
```

Reply:
I like it!